homeassistant/components/sabnzbd/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Sabnzbd",
```

Comment:
Titles that are the same as manifest will automatically be added. This is not needed.
```suggestion
```